src/parseFromAnchor.js

```diff
@@ -7,6 +7,11 @@
 function parseFromAnchor(opt_uri) {
 	var link = document.createElement('a');
 	link.href = opt_uri;
+
+	if(link.protocol === ':' || !/:/.test(link.href)) {
```

Comment:
Should we store this regex as a variable in case the function is called multiple times? So that the regex doesn't need to be re-created every time.